### ISSUE TITLE
Sticky-spam toggle

### ISSUE THREAD

**Issue by SylveonBottle** [Fedoraware/Fedoraware#831]
Yo can someone write this real quick?

Just a thing where if you hold [Button] it launches stickies the moment it's possible.
Very useful for sticky-spamming a large group of enemies without spamming manually and giving yourself arthritis. 

Optional: it'd be cool to set a percentage to charge before shooting. For example, if we put it at 10%, it'll charge every sticky to 10% first then shoot and immediately start charging the next one.

Basically, just a sticky spam button. It was literally one of my favorite features of Rijin since sticky-spamming and holding rightclick is super OP with kritz, without constantly needing to click-click-click and mess up your aim with your finger movements.
